hoomd/md/methods.py

```diff
@@ -51,9 +51,17 @@ class NVT(_Method):
     where :math:`g` is the number of degrees of freedom, and :math:`k_B T_0` is
     the set point (*kT* above).
 
-    .. rubric:: Rotational degrees of freedom
+    .. note::
 
-    `NVT` integrates rotational degrees of freedom. #TODO
+        Coupling constant `tau` in Nosé-Hoover thermostat should be set within 
+        reasonable range to avoid abrupt fluctuation in temperature in case of 
+        small `tau` , also to avoid long time to equilibrate in case of large 
+        `tau`. Recommended value for most of systems is 100 time steps. In time 
+        units, 100 * ``dt``, where ``dt`` is the length of the time step.
```

Comment:
I would prefer just recommending `100 * dt` rather than the notation of 100 time steps.